test/test_ops.py

```diff
@@ -0,0 +1,38 @@
+# Copyright Contributors to the Pyro project.
+# SPDX-License-Identifier: Apache-2.0
+
+import weakref
+
+import pytest
+
+from funsor.distribution import BACKEND_TO_DISTRIBUTIONS_BACKEND
+from funsor.util import get_backend
+from funsor.ops import WrappedTransformOp
+
+
+@pytest.fixture
+def dist():
+    try:
+        module_name = BACKEND_TO_DISTRIBUTIONS_BACKEND[get_backend()]
+    except KeyError:
+        pytest.skip(f"missing distributions module for {get_backend()}")
+    return pytest.importorskip(module_name).dist
+
+
+def test_transform_op_cache(dist):
+    t = dist.transforms.PowerTransform(0.5)
+    W = WrappedTransformOp
+    assert W(t) is W(t)
+    assert W(t).inv is W(t).inv
```

Comment:
Looks like this test is failing on the jax backend because `_InverseTransform` isn't cached in NumPyro?